src/tsconfig.app.json

```diff
@@ -12,7 +12,7 @@
     "secrets.ts"
   ],
   "include": [
-    "/src/**/*.d.ts"
+    "**/*.d.ts"
```

Comment:
Note that this is loading from inside `src` since tsconfig.app.json is located there!